src/App.css

```diff
@@ -27,5 +27,5 @@
 
 body {
   background-color: var(--color-primary);
-  color: var(--color-white);
+  /* color: var(--color-white); */
```

Comment:
before push remove comment

Reply:
Thanks for the review and yes will remove that.